site/src/components/gradient-card.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Gradient card
+description: The gradient card component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - gradient-card
+---
+
+# Gradient card
+
+The `.gradient-card` class contains a predefined gradient border, default border width and default border radius. The inner `.gradient-card-content` element has a default background color and padding but can be modified by Atomic classes.
+
+```html
+<div class="gradient-card tag border-radius-rounded">
+	<div class="gradient-card-content">
+		<p>Gradient card default</p>
+	</div>
+</div>
+```
+
+```html
+<div class="gradient-card">
+	<div class="gradient-card-content background-color-body-medium padding-md">
+		<p>Gradient card with atomics</p>
+	</div>
+</div>
+```
+
+## Usage
+
+Gradient cards are used to bring attention to a section or for notification messages. Here is a common use case.
+
+```html
+<div class="gradient-card">
+	<div class="gradient-card-content">
+		<p>
+			<span class="icon border border-color-primary margin-right-xxs"></span>
```

Comment:
Is it a common pattern for gradient-card to have an icon? If so, the icon needs to be vertically centered with the text next to it, otherwise remove it from your example.

![image](https://github.com/microsoft/atlas-design/assets/57374379/2bfb692b-76da-4b6d-b8de-b6109836b4e5)

Take a look at the `notification` component implementation as a reference.

Reply:
I see on the screenshots you provided the icon usage, so it's alignment have to be fixed.

I'd recommend creating `gradient-card-title` (exact same thing that's done for notification - https://design.learn.microsoft.com/pulls/533/components/notification.html#notification-with-title) which would hold the CSS so the final markup structure will be like this:

```html
<div class="gradient-card">
    <div class="gradient-card-content">
        <p class="gradient-card-title">
            <span class="icon" aria-hidden="true">
                <svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
                    <path d="M4.39804 9.80687C4.57428 9.93117 4.78476 9.99773 5.00043 9.99735C5.21633 9.99751 5.42686 9.93001 5.60243 9.80435C5.77993 9.67387 5.91464 9.49356 5.98943 9.28635L6.43643 7.91335C6.55086 7.56906 6.74391 7.25615 7.00028 6.99943C7.25665 6.74272 7.56929 6.54924 7.91343 6.43435L9.30443 5.98335C9.45636 5.92899 9.59364 5.84019 9.70551 5.7239C9.81738 5.60762 9.9008 5.467 9.94924 5.31308C9.99767 5.15916 10.0098 4.99611 9.98468 4.83672C9.95955 4.67732 9.89786 4.52591 9.80443 4.39435C9.67034 4.20842 9.47939 4.07118 9.26043 4.00335L7.88543 3.55635C7.54091 3.44212 7.22777 3.24915 6.97087 2.99277C6.71396 2.73638 6.52035 2.42363 6.40543 2.07935L5.95343 0.691348C5.88113 0.489044 5.74761 0.314283 5.57143 0.191348C5.43877 0.097621 5.28607 0.03614 5.12548 0.0117941C4.96489 -0.0125515 4.80083 0.000909116 4.64636 0.0511065C4.49188 0.101304 4.35125 0.186853 4.23564 0.30095C4.12004 0.415047 4.03265 0.554544 3.98043 0.708348L3.52343 2.10835C3.40884 2.44317 3.21967 2.74758 2.97022 2.9986C2.72076 3.24962 2.41753 3.44067 2.08343 3.55735L0.692428 4.00535C0.540653 4.05974 0.403522 4.14852 0.291767 4.26474C0.180011 4.38095 0.0966621 4.52145 0.0482407 4.67523C-0.000180673 4.82902 -0.0123605 4.99192 0.0126534 5.1512C0.0376676 5.31048 0.0991972 5.46181 0.192428 5.59335C0.320272 5.7728 0.501046 5.90777 0.709428 5.97935L2.08343 6.42435C2.52354 6.57083 2.90999 6.84518 3.19343 7.21235C3.35585 7.42298 3.4813 7.65968 3.56443 7.91235L4.01643 9.30335C4.08846 9.50664 4.22179 9.68257 4.39804 9.80687ZM4.48343 2.39235L5.01043 1.01535L5.44943 2.39235C5.61312 2.8855 5.88991 3.33351 6.25767 3.70058C6.62544 4.06765 7.07397 4.34359 7.56743 4.50635L8.97343 5.03535L7.59143 5.48335C7.09866 5.64764 6.65095 5.92451 6.28382 6.29198C5.9167 6.65945 5.64026 7.10742 5.47643 7.60035L4.95343 8.97835L4.50443 7.59935C4.34335 7.10608 4.06943 6.65718 3.70443 6.28835C3.3356 5.92031 2.88653 5.64272 2.39243 5.47735L1.01443 4.95535L2.40043 4.50535C2.88672 4.33672 3.32775 4.05855 3.68943 3.69235C4.04901 3.32464 4.32049 2.88016 4.48343 2.39235ZM10.5353 13.8494C10.6713 13.9456 10.8337 13.9973 11.0003 13.9974C11.1654 13.9974 11.3264 13.9464 11.4613 13.8514C11.6008 13.7529 11.7058 13.6129 11.7613 13.4514L12.0093 12.6894C12.0625 12.5309 12.1515 12.3869 12.2693 12.2684C12.3867 12.1498 12.5307 12.0611 12.6893 12.0094L13.4613 11.7574C13.619 11.7029 13.7557 11.6004 13.8523 11.4644C13.9257 11.3614 13.9736 11.2424 13.9921 11.1173C14.0106 10.9922 13.9992 10.8645 13.9588 10.7447C13.9184 10.6249 13.8501 10.5163 13.7597 10.428C13.6692 10.3396 13.5591 10.2739 13.4383 10.2364L12.6743 9.98738C12.5162 9.93481 12.3724 9.84619 12.2544 9.72852C12.1364 9.61086 12.0473 9.46736 11.9943 9.30938L11.7423 8.53638C11.6886 8.37853 11.586 8.24191 11.4493 8.14638C11.3473 8.07343 11.2295 8.02549 11.1056 8.00642C10.9816 7.98736 10.8549 7.99772 10.7357 8.03665C10.6164 8.07558 10.508 8.142 10.4192 8.23054C10.3304 8.31909 10.2636 8.42727 10.2243 8.54638L9.97731 9.30838C9.92502 9.46602 9.83747 9.60967 9.72131 9.72838C9.60657 9.84485 9.46665 9.93345 9.31231 9.98738L8.53931 10.2394C8.38025 10.2933 8.2422 10.3959 8.1447 10.5326C8.04721 10.6694 7.99522 10.8333 7.99611 11.0013C7.99699 11.1692 8.0507 11.3326 8.14963 11.4683C8.24856 11.604 8.38769 11.7051 8.54731 11.7574L9.31031 12.0044C9.46917 12.0578 9.61358 12.1471 9.73231 12.2654C9.85053 12.3836 9.93896 12.5283 9.99031 12.6874L10.2433 13.4614C10.2981 13.6178 10.4001 13.7534 10.5353 13.8494ZM9.62231 11.0564L9.44331 10.9974L9.62731 10.9334C9.92907 10.8284 10.2027 10.6556 10.4273 10.4284C10.6537 10.1994 10.8248 9.92164 10.9273 9.61638L10.9853 9.43838L11.0443 9.61938C11.1463 9.92608 11.3185 10.2047 11.5471 10.4332C11.7757 10.6617 12.0545 10.8336 12.3613 10.9354L12.5563 10.9984L12.3763 11.0574C12.0689 11.1596 11.7898 11.3322 11.5611 11.5616C11.3324 11.791 11.1606 12.0707 11.0593 12.3784L11.0003 12.5594L10.9423 12.3784C10.8409 12.0702 10.6687 11.7901 10.4394 11.5605C10.2102 11.3309 9.93033 11.1583 9.62231 11.0564Z" fill="url(#paint0_linear_2541_142083)"/>
                    <defs>
                        <linearGradient id="paint0_linear_2541_142083" x1="0" y1="0" x2="14.5813" y2="0.0896041" gradientUnits="userSpaceOnUse">
                            <stop stop-color="#B84DC6"/>
                            <stop offset="1" stop-color="#3578CD"/>
                        </linearGradient>
                    </defs>
                </svg>
            </span>
            <span>Title</span>
        </p>
        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus vitae eleifend augue. Sed a neque tincidunt, pretium nisl vitae, consectetur lacus <a href="#">Link</a>.</p>
    M/div>
</div>
```

Reply:
I didn't see this comment originally but was looking at notifications styles like you mentioned and did exactly that. I like this svg implementation better so I've updated that as well.